tcms/testplans/static/testplans/js/get.js

```diff
@@ -358,6 +359,26 @@ function toolbarEvents() {
         }
 
     });
+
+    $('#delete_button').click(function(ev) {
+        let selectedCases = getSelectedTestCases();
+
+        if (!selectedCases.length) {
```

Comment:
Should ask the `Are you sure` question, see the bulk-delete button in refactor_testrun_ui branch.